webapp/metrics/helper.py

```diff
@@ -39,10 +39,11 @@ def build_metrics_json(
 
     if metric_bucket == 'd':
         start = end - relativedelta.relativedelta(
-            days=metric_period)
+            days=metric_period - 1)
```

Comment:
Thanks for moving it into logic. Would be good to keep a comment what explains the reasoning of `-1` for future generations.

Reply:
Added :+1: